### PR TITLE
SAK-489 add draft method for fetching articles

### PR DESCRIPTION
**Pull request by rafalkanski**
@arkadiuszmalina @bartosz-kahl

### CODE REVIEW

File: spec/Snt/Capi/Entity/ArticleSpec.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace spec\Snt\Capi\Entity;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
```
Comment (arkadiuszmalina) [inline spec/Snt/Capi/Entity/ArticleSpec.php]: do you need this?

File: spec/Snt/Capi/Repository/ArticleRepositorySpec.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace spec\Snt\Capi\Repository;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
```
Comment (arkadiuszmalina) [inline spec/Snt/Capi/Repository/ArticleRepositorySpec.php]: as above

File: spec/Snt/Capi/Entity/ArticleSpec.php (hunk 1 above)
Comment (rafalkanski) [inline spec/Snt/Capi/Entity/ArticleSpec.php]: :(

File: spec/Snt/Capi/ClientSpec.php [hunk 3]
```diff
@@ -5,6 +5,7 @@
 use PhpSpec\ObjectBehavior;
 use Prophecy\Argument;
```
Comment (arkadiuszmalina) [inline spec/Snt/Capi/ClientSpec.php]: probably here as well

File: spec/Snt/Capi/Entity/ArticleSpec.php (hunk 1 above)
Comment (rafalkanski) [inline spec/Snt/Capi/Entity/ArticleSpec.php]: chyba ide do zwolnienia za takie karygodne bledy :(

**Comment by arkadiuszmalina** [comment]
👍

**Comment by barkahl** [comment]
👍

**Pull request merged**